scripts/installers/Dockerfile.ngramtools

```diff
@@ -0,0 +1,13 @@
+# How to use? Build it from NeMo root folder:
```

Comment:
Do we need license header for this file?

Reply:
added